res/layout/card_layout.xml

```diff
@@ -52,6 +52,20 @@
                         android:textSize="@dimen/tv_menu_font_size" />
                 </LinearLayout>
             </RelativeLayout>
+            
```

Comment:
I think this is a nice idea to populate the info card with more valuable information but I would add more info on codecs first then hwdec or swdec without details.